.github/pull_request_template.md

```diff
@@ -1,7 +1,35 @@
+## Description
+<!-- Provide a brief description of the changes in this PR -->
+
+## Related Issue
+
 This PR implements issue(s) #
```

Comment:
```suggestion
This PR addresses/relates to issue(s) #
```
I suggest that as PR may not necessarily implement the issue (many times a feature is implemented in multiple PRs)